packages/hooks/src/use-field-id/use-field-id.ts

```diff
@@ -4,7 +4,7 @@ import { getFieldId } from '@commercetools-uikit/utils';
 type CreateIdFn = () => string;
 
 const useFieldId = (id: string | undefined, createIdFn: CreateIdFn): string => {
```

Comment:
🤔 Can't one argument be `type TCreateIDFn = () => string | undefined`? 

Feels like we add this additional argument to not have changes spread out too much in the code base and be able to pass the return of `createSequentialId` in, right?

Reply:
Disregard, I get it now. We pass the ID or use the return of the incrementing fn.